src/feature/nodelist/nodelist.c

```diff
@@ -2501,7 +2501,7 @@ compute_frac_paths_available(const networkstatus_t *consensus,
    * building exit paths */
   /* Update our understanding of whether the consensus has exits */
   consensus_path_type_t old_have_consensus_path = have_consensus_path;
-  have_consensus_path = ((np > 0) ?
+  have_consensus_path = ((nu > 0) ?
```

Comment:
We've switched this check a few times, and it breaks different things each time.
So we need to be very careful about this change.
I'm not sure if it is correct.
Please check the history of changes to this line.

We also need to change variable names, log messages, and documentation, see:
https://trac.torproject.org/projects/tor/ticket/27448#comment:3